src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -121,34 +121,34 @@
     <value>Missing '{0}' attribute.</value>
   </data>
   <data name="MissingDiagnosticAnalyzerAttributeTitle" xml:space="preserve">
-    <value>Missing diagnostic analyzer attribute.</value>
+    <value>Missing 'DiagnosticAnalyzer' attribute.</value>
   </data>
   <data name="MissingDiagnosticAnalyzerAttributeDescription" xml:space="preserve">
-    <value>Non-abstract sub-types of DiagnosticAnalyzer should be marked with DiagnosticAnalyzerAttribute(s). The argument to this attribute(s), if any, determine the supported languages for the analyzer. Analyzer types without this attribute will be ignored by the analysis engine.</value>
+    <value>Non-abstract sub-types of DiagnosticAnalyzer should be marked with DiagnosticAnalyzer attribute(s). The argument to this attribute(s), if any, determine the supported languages for the analyzer. Analyzer types without this attribute will be ignored by the analysis engine.</value>
   </data>
   <data name="AddLanguageSupportToAnalyzerMessage" xml:space="preserve">
-    <value>Diagnostic analyzer '{0}' may be able to support both C# and Visual Basic. Consider adding an argument to DiagnosticAnalyzerAttribute for '{1}' language support.</value>
+    <value>Diagnostic analyzer '{0}' may be able to support both C# and Visual Basic. Consider adding an argument to 'DiagnosticAnalyzer' attribute for '{1}' language support.</value>
   </data>
   <data name="AddLanguageSupportToAnalyzerTitle" xml:space="preserve">
     <value>Recommend adding language support to diagnostic analyzer.</value>
   </data>
   <data name="AddLanguageSupportToAnalyzerDescription" xml:space="preserve">
-    <value>Diagnostic analyzer is marked as supporting only one language, but the analyzer assembly doesn't seem to refer to any language specific CodeAnalysis assemblies, and so is likely to work for more than one language. Consider adding an additional language argument to DiagnosticAnalyzerAttribute.</value>
+    <value>Diagnostic analyzer is marked as supporting only one language, but the analyzer assembly doesn't seem to refer to any language specific CodeAnalysis assemblies, and so is likely to work for more than one language. Consider adding an additional language argument to 'DiagnosticAnalyzer' attribute.</value>
   </data>
   <data name="ApplyDiagnosticAnalyzerAttribute_1" xml:space="preserve">
-    <value>Apply DiagnosticAnalyzer attribute for '{0}'.</value>
+    <value>Apply 'DiagnosticAnalyzer' attribute for '{0}'.</value>
```

Comment:
💭 Another option here is:

```suggestion
    <value>Apply [DiagnosticAnalyzer] for '{0}'.</value>
```

Reply:
I was thinking about this syntax first but then I thought that was too C# oriented (thinking about vb.net devs). Let me know if you still prefer to go for the brackets.